runtime/jcl/common/reflecthelp.c

```diff
@@ -453,7 +456,7 @@ classForSignature(struct J9VMThread *vmThread, U_8 **sigDataPtr, struct J9ClassL
 
 	/* Non-array case */
 	switch (c) {
-	case 'L': {
+	case 'L': { /* TODO: ValueTypes Once it is known if Q and L types share a ramClass, revisit this. See https://github.com/eclipse/openj9/issues/4083 */
```

Comment:
Formatting - more readable to put the long comment before the code than to append it to the same line